docs/Editing-Configuration-Files.md

```diff
@@ -91,6 +91,7 @@ Here is a sample of the three basic types: respectively Boolean, Number and Stri
  * **torrent-added-verify-mode:** String ("fast", "full", default: "fast") Whether newly-added torrents' local data should be fully verified when added, or wait and verify them on-demand later. See [#2626](https://github.com/transmission/transmission/pull/2626) for more discussion.
  * **utp-enabled:** Boolean (default = true) Enable [Micro Transport Protocol (µTP)](https://en.wikipedia.org/wiki/Micro_Transport_Protocol)
  * **preferred-transport:** String ("utp" = Prefer µTP, "tcp" = Prefer TCP; default = "utp") Choose your preferred transport protocol (has no effect if one of them is disabled).
+ * **sleep-per-seconds-during-verify:** Number (default = 100) Controls the duration in milliseconds for which the verification process will pause to reduce disk I/O pressure.
```

Comment:
What would you think of:

- naming the settings.json quark `verify-sleep-interval-msec`
- naming the tr_session_settings property `std::chrono::milliseconds verify_sleep_interval`
- having the tr_verifier method be `set_sleep_interval(std::chrono::milliseconds)`

I like this proposed name better than the current one, but I'm not 100% sold on either name :man_shrugging:

Reply:
I think that the current name is a little bit too long, I've just kept it as it was before my PR to avoid any confusion. The only problem I see with your proposal is that the property gets a bit less self-explanatory, namely at first glance it might not be clear that the sleep interval is applied every second (still, this is clearly stated inside the doc). If you think that my point is irrelevant, I will proceed immediately into change it 👍.

Reply:
Ehh, I'm not going to block the  PR on this change. Like I said, I'm not 100% sure that my suggestion is The Right Name either.